### PR TITLE
Another bunch of urls  in docstrings

### PR DESCRIPTION
**Pull request by mmatera**
…ogic, comparison, datentime, attributes and binary

### CODE REVIEW

File: mathics/builtin/datentime.py [hunk 1]
```diff
@@ -989,6 +1005,8 @@ def evaluate(self, evaluation):
 
 class EasterSunday(Builtin):  # Calendar`EasterSunday
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/EasterSunday.html</url>
```
Comment (TiagoCavalcante) [inline mathics/builtin/datentime.py:1008]: This link is broken

File: mathics/builtin/datentime.py [hunk 2]
```diff
@@ -1059,6 +1079,8 @@ def apply(self, n, evaluation):
 
 class SystemTimeZone(Predefined):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/SystemTimeZone.html</url>
```
Comment (TiagoCavalcante) [inline mathics/builtin/datentime.py:1082]: This link is broken

File: mathics/builtin/datentime.py [hunk 3]
```diff
@@ -1164,6 +1190,9 @@ def apply_3(self, expr, t, failexpr, evaluation):
 
 class TimeZone(Predefined):
     """
+    <url>:Time Zone:https://en.wikipedia.org/wiki/Time_zone</url>
```
Comment (TiagoCavalcante) [inline mathics/builtin/datentime.py:1193]: I think a backslash is missing here

File: mathics/builtin/evaluation.py [hunk 4]
```diff
@@ -10,6 +10,8 @@
 
 class RecursionLimit(Predefined):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/RecursionLimit.html</url>
```
Comment (TiagoCavalcante) [inline mathics/builtin/evaluation.py:13]: The link should be https://reference.wolfram.com/language/ref/$RecursionLimit.html

File: mathics/builtin/layout.py [hunk 5]
```diff
@@ -162,6 +172,8 @@ class Left(Builtin):
 
 class NonAssociative(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/NonAssociative.html</url>
```
Comment (TiagoCavalcante) [inline mathics/builtin/layout.py:175]: This link is broken

File: mathics/builtin/layout.py [hunk 6]
```diff
@@ -197,6 +211,8 @@ class Postfix(BinaryOperator):
 
 class Precedence(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Precedence.html</url>
```
Comment (TiagoCavalcante) [inline mathics/builtin/layout.py:214]: This link is broken

File: mathics/builtin/logic.py [hunk 7]
```diff
@@ -351,6 +367,8 @@ def apply(self, args, evaluation):
 
 class True_(Predefined):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/True_.html</url>
```
Comment (TiagoCavalcante) [inline mathics/builtin/logic.py:370]: The link should be https://reference.wolfram.com/language/ref/True.html

**Review by TiagoCavalcante — CHANGES_REQUESTED** [review]
LGTM - just a few things

File: mathics/builtin/layout.py (hunk 5 above)
Comment (mmatera) [reply mathics/builtin/layout.py:175]: That is funny, but it is a builtin symbol in WMA, but does not have an entry in WR....

File: mathics/builtin/layout.py (hunk 6 above)
Comment (mmatera) [reply mathics/builtin/layout.py:214]: Same as before...

File: mathics/builtin/datentime.py (hunk 1 above)
Comment (mmatera) [reply mathics/builtin/datentime.py]: Fixed

File: mathics/builtin/arithfns/basic.py [hunk 8]
```diff
@@ -143,6 +145,8 @@ def eval(self, n, evaluation):
 
 class Divide(BinaryOperator):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Divide.html</url>
+
     <dl>
     <dt>'Divide[$a$, $b$]'
```
Comment (rocky) [inline mathics/builtin/arithfns/basic.py:151]: dt indent

File: mathics/builtin/arithfns/basic.py [hunk 9]
```diff
@@ -207,6 +211,8 @@ class Divide(BinaryOperator):
 
 class Minus(PrefixOperator):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Minus.html</url>
+
     <dl>
     <dt>'Minus[$expr$]'
```
Comment (rocky) [inline mathics/builtin/arithfns/basic.py:217]: dt indent

File: mathics/builtin/arithfns/basic.py [hunk 10]
```diff
@@ -632,6 +642,8 @@ def eval_check(self, x, y, evaluation):
 
 class Sqrt(SympyFunction):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Sqrt.html</url>
+
     <dl>
     <dt>'Sqrt[$expr$]'
```
Comment (rocky) [inline mathics/builtin/arithfns/basic.py:648]: dt indent

File: mathics/builtin/arithfns/basic.py [hunk 11]
```diff
@@ -632,6 +642,8 @@ def eval_check(self, x, y, evaluation):
 
 class Sqrt(SympyFunction):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Sqrt.html</url>
```
Comment (rocky) [inline mathics/builtin/arithfns/basic.py:645]: Long line, split after "WMA link".

File: mathics/builtin/arithfns/sums.py [hunk 12]
```diff
@@ -11,6 +11,8 @@
 
 class Accumulate(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Accumulate.html</url>
```
Comment (rocky) [inline mathics/builtin/arithfns/sums.py:14]: Long line, split after "WMA link"

File: mathics/builtin/arithfns/sums.py [hunk 13]
```diff
@@ -11,6 +11,8 @@
...
+
     <dl>
     <dt>'Accumulate[$list$]'
```
Comment (rocky) [inline mathics/builtin/arithfns/sums.py:17]: dt indent

File: mathics/builtin/arithfns/sums.py [hunk 14]
```diff
@@ -26,6 +28,8 @@ class Accumulate(Builtin):
 
 class Total(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Total.html</url>
+
     <dl>
     <dt>'Total[$list$]'
```
Comment (rocky) [inline mathics/builtin/arithfns/sums.py:34]: dt indent

File: mathics/builtin/arithmetic.py [hunk 15]
```diff
@@ -265,6 +266,8 @@ class Abs(_MPMathFunction):
 
 class Arg(_MPMathFunction):
     """
+    <url>:Arg:https://en.wikipedia.org/wiki/Argument_(complex_analysis)</url> \
+    (<url>:WMA link:https://reference.wolfram.com/language/ref/Arg.html</url>)
```
Comment (rocky) [inline mathics/builtin/arithmetic.py:270]: Remove "WMA link" -> "WMA" since in parenthesis.

File: mathics/builtin/arithmetic.py [hunk 16]
```diff
@@ -569,6 +581,9 @@ def to_sympy(self, expr, **kwargs):
 
 class Conjugate(_MPMathFunction):
     """
+    <url>:Complex Conjugate:https://en.wikipedia.org/wiki/Complex_conjugate</url> \
+    (<url>:WMA link:https://reference.wolfram.com/language/ref/Conjugate.html</url>)
```
Comment (rocky) [inline mathics/builtin/arithmetic.py:585]: Remove "link" - in parenthesis

File: mathics/builtin/arithmetic.py [hunk 17]
```diff
@@ -569,6 +581,9 @@ def to_sympy(self, expr, **kwargs):
 
 class Conjugate(_MPMathFunction):
     """
+    <url>:Complex Conjugate:https://en.wikipedia.org/wiki/Complex_conjugate</url> \
```
Comment (rocky) [inline mathics/builtin/arithmetic.py:584]: Split after "Complex Conjugate" - line too long.

File: mathics/builtin/arithmetic.py [hunk 18]
```diff
@@ -603,6 +618,8 @@ class Conjugate(_MPMathFunction):
 
 class DirectedInfinity(SympyFunction):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/DirectedInfinity.html</url>
```
Comment (rocky) [inline mathics/builtin/arithmetic.py:621]: Line too long.

File: mathics/builtin/arithmetic.py [hunk 19]
```diff
@@ -714,6 +734,8 @@ def evaluate(self, evaluation):
 
 class Im(SympyFunction):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Im.html</url>
```
Comment (rocky) [inline mathics/builtin/arithmetic.py:737]: remove "link"

File: mathics/builtin/arithmetic.py [hunk 20]
```diff
@@ -694,6 +711,9 @@ def to_sympy(self, expr, **kwargs):
 
 class I(Predefined):
     """
+    <url>:Imaginary unit:https://en.wikipedia.org/wiki/Imaginary_unit</url> \
+    (<url>:WMA link:https://reference.wolfram.com/language/ref/I.html</url>)
```
Comment (rocky) [inline mathics/builtin/arithmetic.py:715]: remove "link"

File: mathics/builtin/arithmetic.py [hunk 21]
```diff
@@ -1175,6 +1213,8 @@ class Real_(Builtin):
 
 class RealNumberQ(Test):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/RealNumberQ.html</url>
```
Comment (rocky) [inline mathics/builtin/arithmetic.py:1216]: This function is not in WMA. Comment out.

File: mathics/builtin/assignments/assign_binaryop.py [hunk 22]
```diff
@@ -50,6 +52,8 @@ class AddTo(BinaryOperator):
 
 class Decrement(PostfixOperator):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Decrement.html</url>
```
Comment (rocky) [inline mathics/builtin/assignments/assign_binaryop.py:55]: Line too long

File: mathics/builtin/assignments/assign_binaryop.py [hunk 23]
```diff
@@ -142,6 +150,8 @@ class Increment(PostfixOperator):
 
 class PreIncrement(PrefixOperator):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/PreIncrement.html</url>
+
     <dl>
     <dt>'PreIncrement[$x$]'
```
Comment (rocky) [inline mathics/builtin/assignments/assign_binaryop.py:156]: dt indent

File: mathics/builtin/assignments/assign_binaryop.py [hunk 24]
```diff
@@ -196,6 +208,8 @@ class PreDecrement(PrefixOperator):
 
 class SubtractFrom(BinaryOperator):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/SubtractFrom.html</url>
```
Comment (rocky) [inline mathics/builtin/assignments/assign_binaryop.py:211]: line too long

File: mathics/builtin/assignments/assign_binaryop.py [hunk 25]
```diff
@@ -196,6 +208,8 @@ class PreDecrement(PrefixOperator):
...
+
     <dl>
     <dt>'SubtractFrom[$x$, $dx$]'
```
Comment (rocky) [inline mathics/builtin/assignments/assign_binaryop.py:214]: dt indent

File: mathics/builtin/assignments/assignment.py [hunk 26]
```diff
@@ -145,6 +148,8 @@ def apply(self, lhs, rhs, evaluation):
 
 class SetDelayed(Set):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/SetDelayed.html</url>
```
Comment (rocky) [inline mathics/builtin/assignments/assignment.py:151]: line too long

File: mathics/builtin/assignments/types.py [hunk 27]
```diff
@@ -77,6 +81,8 @@ def apply(self, symbol, evaluation):
 
 class NValues(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/NValues.html</url>
```
Comment (rocky) [inline mathics/builtin/assignments/types.py:84]: Not in WMA

File: mathics/builtin/assignments/upvalues.py [hunk 28]
```diff
@@ -15,6 +15,8 @@
 
 class UpSet(BinaryOperator, _SetOperator):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/UpSet.html</url>
```
Comment (rocky) [inline mathics/builtin/assignments/upvalues.py:18]: Not showing in docs because module has no comment. Moved into "assignment".

**Pull request merged**